addons/binding/org.openhab.binding.smappee/src/main/java/org/openhab/binding/smappee/service/SmappeeAccessTokenResponse.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.openhab.binding.smappee.service;
+
+/**
+ * Interface for readings updates
+ *
+ * @author Niko Tanghe
+ */
+
+// {"access_token":"92dXXX49-0645-39ae-b7be-fXXXX65397ed","refresh_token":"8bXXXX54-4a8e-362d-a59a-d7bXXXXX4fd0","expires_in":86400}
+public class SmappeeAccessTokenResponse {
+    public String access_token;
```

Comment:
fields should not have underscore in name.

Reply:
is for 1 on 1 mapping with the JSON